plugins/com.google.cloud.tools.eclipse.appengine.flex/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -4,5 +4,4 @@ org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.7
 org.eclipse.jdt.core.compiler.compliance=1.7
 org.eclipse.jdt.core.compiler.problem.assertIdentifier=error
 org.eclipse.jdt.core.compiler.problem.enumIdentifier=error
-org.eclipse.jdt.core.compiler.problem.forbiddenReference=warning
```

Comment:
It will be "error" by default if you haven't changed the global error/warning setting. We shouldn't have project-specific error/waning customization.